source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -297,107 +576,251 @@ msgid "History"
+msgid ""
+"The dojot has an internal *Certificate Authority* (`CA`_) capable of "
+"issuing x.509 certificates so that devices can communicate with the "
+"platform through a secure channel (using the TLS protocol). When "
+"requesting a certificate for the platform, it is necessary to inform a "
+"`CSR`_, which will go through a series of validations until arriving at "
+"the internal Certificate Authority, which, in turn, if all checks pass "
+"successfully, will sign a certificate and link this certificate to the "
+"device registration. The ``x509-identity-mgmt`` component is responsible "
+"for providing certificate-related services for devices."
+msgstr ""
+"A plataforma dojot possui internamente uma autoridade certificadora "
+"(`CA`_) capaz de emitir certificados x.509 para que os dispositivos "
+"possam se comunicar com a plataforma através de um canal seguro (usando o"
+" protocolo TLS). Ao requisitar um certificado para a plataforma, é "
+"necessário informar um `CSR`_, o qual passará por uma série de validações"
+" até chegar na Autoridade Certificadora interna, que por sua vez, se "
+"todas as verificações passarem com sucesso, assinará um certificado e "
+"vinculará este certificado ao registro do dispositivo. O componente "
+"``x509-identity-mgmt`` é responsável por oferecer os serviços "
+"relacionados a certificados para dispositivos."
 
-#: ../../source/internal-communication.rst:556
+#: ../../source/internal-communication.rst:558
 msgid "Kafka-WS"
 msgstr ""
 
-#: ../../source/internal-communication.rst:558
-msgid "*Kafka WS* service allows users to retrieve conditional and/or partial real time data from a given dojot topic in a Kafka cluster. It works with pure websocket connections, so you can create websocket clients in any language you want as long as they support RFC 6455."
+#: ../../source/internal-communication.rst:560
+#, fuzzy
+msgid ""
+"*The Kafka WS* service allows users to retrieve conditional and/or "
+"partial real time data from a given dojot topic in a Kafka cluster. It "
+"works with pure websocket connections, so you can create websocket "
+"clients in any language you want as long as they support RFC 6455."
 msgstr ""
+"O serviço *Kafka WS* permite aos usuários a recuperação condicional e/ou "
+"parcial de dados em tempo real de um determinado tópico da dojot em um "
+"Cluster Kafka. Funciona com uma conexão websocket puro, para que clientes"
```

Comment:
As in another comment in this same review, I wrongly corrected the word `pura` to `puro`.